go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	golang.org/x/crypto v0.0.0-20220525230936-793ad666bf5e
 	golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e
 	google.golang.org/genproto v0.0.0-20220519153652-3a47de7e79bd
-	google.golang.org/grpc v1.49.0
+	google.golang.org/grpc v1.50.1
```

Comment:
google.golang.org/grpc has been replaced with v1.33.2. cosmos-sdk has replace removed. In the future, lbm-sdk also needs to remove replace.